app/features/admin/authentication.feature

```diff
@@ -0,0 +1,9 @@
+@mvp
```

Comment:
@Maralsotoudehnia  is saying that this feature is not MVP

Reply:
...really? @Maralsotoudehnia what makes authentication as an administrator different from authentication as an analyst such that the admin story is out of scope but the analyst story is in scope?